.github/workflows/manual.yml

```diff
@@ -17,17 +23,20 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        # os: [ubuntu-latest]
         # need to fix https://github.com/rajeeja/tempestremap/runs/3830784744?check_suite_focus=true#step:11:25 for osx build to work
-        # os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
The checks on this branch still only show builds on Linux. Am I missing something ?

Reply:
In another PR, planing to delete linux.yml and only have manual.yml (renamed to reg.yml or so) to run automatically, once I get this green: https://github.com/rajeeja/tempestremap/runs/4074937743?check_suite_focus=true